src/App.tsx

```diff
@@ -10,37 +11,20 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">Selected tab is Tab 1</h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">
-                Tab 1
-              </a>
-            </li>
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
```

Comment:
to improve readability because selectedTabId is tab not it's id 
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```